src/hooks/useSelectionState/useSelectionState.stories.tsx

```diff
@@ -106,3 +106,52 @@ export const ExpandableTable: React.FunctionComponent = () => {
     </Table>
   );
 };
+
+export const ExternalState: React.FunctionComponent = () => {
+  // import { Checkbox } from '@patternfly/react-core';
```

Comment:
Can you please remove comment?

Reply:
Sorry, yes I can if you want, but I wanted to indicate where Checkbox is imported from in the Storybook example because it doesn't show the actual imports. Maybe that's annoying.

Reply:
Or maybe add Example something above.